sycl/include/CL/__spirv/spirv_vars.hpp

```diff
@@ -168,6 +168,8 @@ namespace __spirv {
 
 // Helper function templates to initialize and get vector component from SPIR-V
 // built-in variables
+// Disable index flipping for SYCL Native CPU
```

Comment:
Thanks for pointing this out, we made this change a while ago and never reconsider it. Yes the idea was to have `__spirv_GlobalSize_x ` return the "unflipped" index on native CPU, but this is probably not necessary since we can handle the index flipping in our PI plug in. I'll make sure that we can handle it in the plug in and undo this change in the header, thank you

Reply:
I've undone our change in the header, thanks again for pointing it out

Reply:
@steffenlarsen am I ok to resolve this thread?

Reply:
Absolutely! Thank you!